dev/jenkins_task_validate_image.sh

```diff
@@ -0,0 +1,624 @@
+#!/bin/bash
+#
+# This script configures and deploys Spinnaker on GCP and runs an integration
+# test suite for validation that manages resources on AWS, GCP, and Kubernetes.
+#
+# The following environment variables are expected to be populated:
+#
+# IMAGE_TO_VALIDATE [String] - The name of the image to validate.
+# IMAGE_PROJECT [String] - The GCE project containing the $IMAGE_TO_VALIDATE.
+# If changing this, you will also need to make sure that the $TESTER_SERVICE_ACCOUNT has permission to View (or Edit) the project so that it can download the image.
+# MY_INSTANCE_NAME [String] - The name of the instance to deploy when testing. Must be unique.
+# PARALLELIZE_TESTS [Boolean] - Run each of the top-level test suites in parallel.
+# MY_CLUSTER_NAME [String] - The name of the Kubernetes cluster to be tested against.
+# AWS_ACCESS_KEY [String] - The value of the AWS access key.
+# AWS_SECRET_KEY [String] - The value of the AWS secret key.
+# JENKINS_PASSWORD [String] - Password to Jenkins instance configured for Spinnaker instance.
+# JENKINS_USER [String] - User for Jenkins instance configured for Spinnaker instance.
+# HOME [String] - Home directory for the user running this script.
+# BASE_ARCHIVE_PATH [String] - Path to GCS bucket for build archival.
```

Comment:
Maybe let this be empty and only archive if it is set.
The data isnt actually used for anything [yet].